### DISCUSSION TITLE
using Regions or Markers with continous annotations

### DISCUSSION THREAD

**Discussion by g-no** [katspaugh/wavesurfer.js#2777] [Q&A] [answered]
I want to use wavesurfer/regions to revise existing annotations of speech. There are **no non-annotated regions**, i.e. each regions end time is equal to the start time of the following region (except for the last, of course). I thought that it might be easier to work with markers (adding regions without an end time), so I wouldn't have to move two boundaries, but just one.

My general question is, if this idea reasonable of if I should better work with end times and add some code that adjusts the start and end times if one of them has been changed?

My aim is to play the region as precisely as possible.

**Comment by katspaugh** [comment]
If you don’t need visual overlays that proper regions provide, then using zero-length markers makes sense.
Moving a neighboring region on resize should be easy too.

**Comment by g-no** [comment]
Thanks to your answer I now know that I do need start and end because I want to use visual overlays. Is it possible to make the start markers dragable but not the end markers (maybe don't even show them)?

**Reply by katspaugh** [reply, marked as answer]
Not at the moment, but we can add this option.

**Reply by g-no** [reply]
That would be cool!

**Reply by katspaugh** [reply]
Actually, I've just published alpha 49 where you can hide the end marker via CSS:
```css
      ::part(region-handle-right) {
        display: none;
      }
```
